ai_genomics/pipeline/gtr/make_gtr_projects.py

```diff
@@ -0,0 +1,177 @@
+
+    # Look at abstracts
+    ai_abstract_terms, genom_abstract_terms = [
+        config[f"gtr_{t}_abstract"] for t in ["ai", "genom"]
+    ]
+
+    ai_projs_abstr, genom_projs_abstr = [
+        set(
+            [
+                gtr_id
+                for gtr_id, abstract_text, title in zip(
+                    projects["id"].values(),
+                    projects["abstractText"].values(),
+                    projects["title"].values(),
+                )
+                if any(  # If any of the abstract terms are in the abstract or title
+                    t in str(abstract_text).lower() + str(title).lower() for t in terms
+                )
+            ]
+        )
+        for terms in [ai_abstract_terms, genom_abstract_terms]
+    ]
+
+    logging.info(f"Projects with AI abstract / title: {len(ai_projs_abstr)}")
+    logging.info(f"Projects with genomics abstract / title: {len(genom_projs_abstr)}")
+    logging.info(
+        f"Projects with AI and genomics abstract / title: {len(ai_projs_abstr & genom_projs_abstr)}"
+    )
+
+    #  Intersection of the two sets
+    unified_projects = list(
+        set(ai_projs_abstr.union(ai_topics) & genom_projs_abstr.union(genomics_topics))
+    )
+
+    ai_genomics_combined = (
+        pd.DataFrame.from_dict(projects)
+        .query(f"id in {unified_projects}")
+        .reset_index(drop=True)
+    )
+
+    logging.info(f"AI and genomics projects combined: {len(ai_genomics_combined)}")
+
+    # Get project examples
+    project_examples = ai_genomics_combined.sample(5)[["title", "abstractText"]].rename(
+        columns={"title": "Project title"}
+    )
+    project_examples["Abstract (truncated)"] = (
+        project_examples["abstractText"].str[:700] + "..."
+    )
+    project_examples = project_examples[["Project title", "Abstract (truncated)"]]
+    project_examples.to_markdown(f"{PROJECT_DIR}/outputs/gtr_examples.md", index=False)
+
+    logging.info(project_examples)
+
+    # Save relevant (AI and genomics, AI, genomics) projects locally and to s3
+    ai_genomics_combined.pipe(camel_case_to_snake_case_col_names).to_json(
+        GTR_INPUTS_DIR / "gtr_ai_genomics_projects.json"
+    )
+
+    projects_df = pipe(pd.DataFrame(projects), camel_case_to_snake_case_col_names)[
+        KEEP_GTR_VARS
+    ]
+
+    projects_df["ai"], projects_df["genomics"], projects_df["ai_genomics"] = [
+        projects_df["id"].isin(list_ids)
+        for list_ids in [
+            ai_projs_abstr,
+            genom_projs_abstr,
+            ai_projs_abstr & genom_projs_abstr,
+        ]
+    ]
+
+    filtered_projects = (
+        projects_df.loc[
+            projects_df[["ai", "genomics", "ai_genomics"]].values.sum(axis=1) > 0
+        ]
+        .reset_index(drop=True)
+        .to_csv(GTR_OUTPUTS_DIR / GTR_PROJ_NAME, index=False)
+    )
+
+    send_output_to_s3(
+        str(GTR_OUTPUTS_DIR / GTR_PROJ_NAME), f"crunchbase/{GTR_PROJ_NAME}"
```

Comment:
you're saving this to the crunchbase directory, not GTR:

```suggestion
        str(GTR_OUTPUTS_DIR / GTR_PROJ_NAME), f"gtr/{GTR_PROJ_NAME}"
```